server/api/question/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+
+class Image(models.Model):
+    id = models.AutoField(primary_key=True)
+    scale = models.IntegerField()
+    jax_text = models.TextField(default="")
+    url = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f'{self.id} {self.url}'
+
+
+class Category(models.Model):
+    id = models.AutoField(primary_key=True)
+    diff_level = models.IntegerField()
+    name = models.CharField(max_length=50)
+    info = models.TextField(default="")
+    parent = models.ForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True)
+    is_comp = models.BooleanField()
+
+    def __str__(self):
+        return f'{self.id} {self.name} {self.info}'
+
+
+class Question(models.Model):
+    id = models.AutoField(primary_key=True)
+    parent = models.ForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True)
+    name = models.CharField(max_length=255)
+    question_text = models.TextField(default="set question text...")
+    description = models.TextField(default="set question description...")
+    category_id = models.ForeignKey(Category, on_delete=models.SET_NULL, null=True, blank=True)
+    created_by = models.ForeignKey('auth.User', on_delete=models.SET_NULL, null=True, blank=True, related_name='question_created')
```

Comment:
related_name='question_created' it bette to be `questions_created`? how do you think of it?

Reply:
question_text = models.TextField(default="set question text...") please put the default question text as ""( empty or null is a better choice I reckon.)